docs: add onboarding calendly links to docs (#14)
kurtosis-tech/kurtosis: docs/docs/quickstart.md

slug: /quickstart
---

> These instructions will give you a brief quickstart of Kurtosis and should take 15 minutes. Get a personalized on-boarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 

Reviewer: Maybe can make it flow a bit better? "If you prefer, you can get a personalized...."

> These instructions will give you a brief quickstart of Kurtosis and should take 15 minutes. Get a personalized on-boarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 

Author: ```suggestion
These instructions will give you a brief quickstart of Kurtosis and should take 15 minutes. If you prefer, schedule a personalized onboarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 
```

- Explore [Kurtosis-provided packages being used in production][kurtosis-managed-packages]
- [Search GitHub for Kurtosis packages in the wild][wild-kurtosis-packages]

:::info

> Get a personalized on-boarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 

Reviewer: Same "on-boarding" comment, here and everywhere

Author: ```suggestion
Get a personalized onboarding session with us [here](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding). 
```
